ui/packages/theme/src/global.css

```diff
@@ -2,6 +2,16 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer components {
+	.gr-gap > * + * {
+		margin-top: 0.5rem;
+	}
+	.gr-form-gap > .form + .form {
+		margin-top: 0;
+		border-top: none;
+	}
+}
+
```

Comment:
@gary149 This is the new tailwind style i mentioned.

Reply:
@apply mt-0 border-t-0
For the same reasons.